packages/bpk-component-content-cards/src/ContentCard/ContentCard.module.scss

```diff
@@ -0,0 +1,91 @@
+/* stylelint-disable selector-class-pattern */
+/* stylelint-disable selector-max-compound-selectors */
+@import '~bpk-mixins/index';
+
+$bpk-spacing-v2: true;
+
+@mixin aspect-ratio($width, $height) {
+  position: relative;
+
+  &::before {
+    content: '';
+    display: block;
+    width: 100%;
+    padding-top: ($height / $width) * 100%;
+  }
+
+  > :first-child {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+  }
+}
```

Comment:
@olliecurtis maybe there's something like this already used in `backpack`?
Or better, we could have a component `<AspectRatio ratio={3/4}>` that we could use to have a container that maintains aspect ratio?